modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase

```diff
@@ -966,10 +966,12 @@ definition {
 	@description = "LPS-94760 - Verify that the time format is following the system language (Example: 03:34 PM for en_US and 15:34 for pt)"
 	@priority = 3
 	test SLAProcessingTimeFormatIsFollowingSystemLanguage {
-		property ci.retries.disabled = "true";
-
 		var slaName = "SLAName";
 
+		WorkflowMetrics.setSLADefinitionsJobInterval(
+			definitionsJobInterval = 1,
+			slaJobInterval = 1);
+
```

Comment:
Like in https://github.com/liferay-bpm-qa/liferay-portal/pull/839
This step is for metrics recalculation. To ensure that the data has been read before asserting at the end of the test.